client/src/pages/Plan/Plan.js

```diff
@@ -61,6 +61,14 @@ const Plan = () => {
         });
     });
 
+    const createElectivePlaceholder = () => {
+        const electivePlaceholders = courseAllocations?.filter((courseAllocation) => !isNaN(courseAllocation.course));
+        const code =
+            Math.max.apply(Math, electivePlaceholders?.length ? electivePlaceholders?.map((elective) => parseInt(elective.course)) : [0]) +
+            1;
```

Comment:
Surely make the `+1` on the same line 😢

Reply:
lol prettier formatted it into that, i tried 😢

Reply:
Wtf does the line do lol? Also, what does `.apply()` do in this case?

Reply:
Never mind, I understand it from this: https://stackoverflow.com/questions/21255138/how-does-the-math-max-apply-work

Apparently you can replace it with: `Math.max(electivePlaceholders?.length ? electivePlaceholders?.map((elective) => parseInt(elective.course)) : [0]) + 1` instead of the `apply`

Reply:
ohh yes my bad, i was meant to remove apply after adding the [0]